### ISSUE TITLE
Numpad: change is not fired on the input in inline mode

### ISSUE THREAD

**Issue by dioslaska** [acidb/mobiscroll#384]
If the numpad is displayed in inline mode, the change event is not triggered on the input when the value is filled in. As a consequence, if using MVVM integrations (Angular, Knockout, etc.), the model value is not updated. (versions 2.x and 3.x).

Possible workaround:

```
var settings = {
    display: 'inline',
    preset: 'decimal',
    onFill: function (ev, inst) {
        inst._preventChange = true;
        $(this).trigger('change');
    }
};
```

**Comment by dioslaska** [comment]
Fixed in 3.0.0-beta3

**Comment by dioslaska** [comment]
Fixed in 2.17.3 as well
